### ISSUE TITLE
Atlas download -- Windows

### ISSUE THREAD

**Issue by FedeClaudi** [SainsburyWellcomeCentre/amap-python#4]
Hey, 

When running amap for the first time in windows it gives an error:

`AttributeError: module 'os' has no attribute 'statvfs'`

I believe that statvfs is not supported on windows. 

An alternative solution using shutil:
```
    drive, _ = os.path.splitdrive(file_path)
    total, used, free = shutil.disk_usage(drive)
    return free / 1024**3
```

Full error stack:
```
(amap) C:\Users\Federico>amap -i stitchedImages_100\2 -o Federico\registration --x 2.231 --y 2.231 --z 5.0
WARNING:root:Atlas does not exist, downloading.
Traceback (most recent call last):
  File "c:\users\federico\.conda\envs\amap\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\federico\.conda\envs\amap\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Federico\.conda\envs\amap\Scripts\amap.exe\__main__.py", line 7, in <module>
  File "c:\users\federico\.conda\envs\amap\lib\site-packages\amap\main.py", line 297, in main
    args = prep_registration(args)
  File "c:\users\federico\.conda\envs\amap\lib\site-packages\amap\main.py", line 283, in prep_registration
    atlas_download.main(args.atlas, args.install_path, args.download_path)
  File "c:\users\federico\.conda\envs\amap\lib\site-packages\amap\download\atlas.py", line 28, in main
    extract_requires=extract_requirements_gb[atlas],
  File "c:\users\federico\.conda\envs\amap\lib\site-packages\amap\download\download.py", line 46, in download
    disk_free_gb(os.path.dirname(download_path)) < download_requires
  File "c:\users\federico\.conda\envs\amap\lib\site-packages\amap\tools\system.py", line 197, in disk_free_gb
    stats = os.statvfs(file_path)
AttributeError: module 'os' has no attribute 'statvfs'
```

Cheers, 
Federico

**Comment by adamltyson** [comment]
Thanks Federico. I've also realised there's (at least) one other thing that needs fixing for Windows support (#5). In the mean time, if you wanted to use amap, it should run happily on our cluster.

**Comment by adamltyson** [comment]
Fixed in 4681fd72ef7d30bb9e3f16b4f93f30711eba2753, and full Windows support in 25e9994d6347710f46efecc63b3baaea387231e9. Still needs to be tested though (see #5).
